tools/ClassGen/InstrBuilder.cpp

```diff
@@ -558,3 +558,8 @@ InstrBuilder &InstrBuilder::addMember(MemberType type,
 
   return addMember(*typeInfo, name);
 }
+
+InstrBuilder &InstrBuilder::addFusedActivation() {
+  return addMember(MEMBER_TYPE_INFO(glow::FusedActivation), "FusedActivation")
```

Comment:
Can you add a comment here explaining the need for `FusedActivationArgs`? E.g. something simple like some activations are parameterized, such as Clip which requires a min and max value.

Reply:
Done.